inventory/internal/store/host_test.go

```diff
@@ -1455,6 +1529,10 @@ func Test_LimitAndOffsetWithMetadataFilter(t *testing.T) {
 	assert.Equal(t, findRes2.TotalElements, int32(99))
 }
 
+// Note the delicacy of handling NULL columns. With a naive NEQ filter, nothing is returned as a SQL NULL is not equal
+// to anything, nor is it unequal to anything.
+// See: https://stackoverflow.com/a/18243804
+// To address this, an additional `OR <> = null` clause can be used.
```

Comment:
Why the comment has moved here?

Reply:
it can still happen if we introduce a new column and we dont handle the UNSPECIFIED scenario. Atm for host does not happen anymore